documentation/modules/ROOT/pages/connectors/postgresql.adoc

```diff
@@ -1892,6 +1892,18 @@ See the {link-prefix}:{link-postgresql-connector}#postgresql-data-types[list of
 Each item (regular expression) must match the fully-qualified `<fully-qualified table>:<a comma-separated list of columns>` representing the custom key. +
 Fully-qualified tables could be defined as _schemaName_._tableName_.
 
+|[[postgresql-publication-autocreate-mode]]<<postgresql-publication-autocreate-mode, `publication.autocreate.mode`>>
+|_all_tables_
+|Determine how creation of a publication should work, the default is all_tables.
+_disabled_ - The connector will not attempt to create a publication at all. The expectation is
+that the user has created the publication up-front. If the publication isn't found to exist upon
+startup, the connector will throw an exception and stop.
+_all_tables_ - This acts like the current behavior where if the publication does not exist the
+connector will create the publication for all tables. If the publication exists, we will use it.
+i.e `CREATE PUBLICATION <publication_name> FOR ALL TABLES;`
+_filtered_ - This would create the publication for matching tables only
```

Comment:
If no publication exists, the connector will create a new publication for all those tables matching the current filter configuration (see table/database whitelist/blacklist properties). If the publication already exists, it will be used.